please_copy_and_adapt/train_model.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+###########################
+# Remove or add parameters to fit your needs. You should change your yaml file
+# accordingly.
+# Change DWIMLAbstractSequences for an implementation of your own model.
+# It should be a child of this abstract class.
+############################
+
+""" Train a model for my favorite experiment"""
+
+import argparse
+import logging
+from os import path
+import yaml
+
+from dwi_ml.training.scripts_utils import *
+from dwi_ml.training.trainer_abstract import DWIMLAbstractSequences
+
+
+def parse_args():
+    p = argparse.ArgumentParser(description=__doc__,
+                                formatter_class=argparse.RawTextHelpFormatter)
+    p.add_argument('parameters_filename',
+                   help='Experiment configuration YAML filename. See '
+                        'please_copy_and_adapt/training_parameters.yaml for an '
+                        'example.')
+
+    arguments = p.parse_args()
+    return arguments
+
+
+def check_all_parameters(conf):
```

Comment:
The `check_all_parameters` method could be renamed to something like `check_training_parameters` and dwell in a e.g. `experiment_parameter_utils.py` script (I'd rename the `script_utils.py` to this name as well or think of a better naming). That would allow re-using the method if you have slightly different `train_model.py` scripts without duplicating the code.